### DISCUSSION TITLE
The power controller for 'smart hot water' has a temperature probe, can it be read on the modbus?

### DISCUSSION THREAD

**Discussion by cralwalker** [WillCodeForCats/solaredge-modbus-multi#609] [Q&A]
The power controller SMRT-HOT-WTR-48-S2 with temperature sensor HOTWTR-SENS-RW-S1 is not that smart.. No actions can be automated based on the sensor output.

Is there any kind of sniffer I can add to the modbus to help identify the powercontroller and temperature sensor coms so that I can automate actions in HomeAssistant instead?

**Comment by WillCodeForCats** [comment]
Modbus is a polling protocol so there's no way to observe changes from an event.

To add something to the integration I need to know the registers and how they describe a device.

On top of that, the Modbus/TCP interface is actually a proxy that the inverter abstracts from internally. It's not a one to one mirror of all data available in the inverter.
